Boards/kav_mega.board.json.inactive

```diff
@@ -35,15 +35,15 @@
     "CanInstallFirmware": true,
     "CanResetBoard": true,
     "DelayAfterFirmwareUpdate": 0,
-    "FirmwareBaseName": "kav_efis_fcu_mega",
+    "FirmwareBaseName": "kav_mega",
```

Comment:
the firmware file has to be renamed too:
https://github.com/elral/MobiFlight-Connector/blob/renaming_KAV_customDevices/firmware/kav_efis_fcu_mega_2_4_4.hex